docs/tutorials/dhis2-docker.md

```diff
@@ -3,312 +3,12 @@ id: dhis2-docker
-distro, you want the local binary directory for userland.
-
-```
-sudo curl -L "https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-```
-
-Mark it as executable after inspecting the binary.
-
-```
-sudo chmod +x /usr/local/bin/docker-compose
-```
-
-Test out the install with:
-
-```
-which docker-compose
-docker-compose --version
-```
-
-## Install Nginx (_Optional_)
-
-The standard package for Nginx is solid and stable.
-
-```
-sudo apt-get update
-sudo apt-get install nginx
-```
-
-### Setup default site configuration
-
-This overwrites the Nginx default site. If you want to preserve that, [add this
-as a new vhost](https://www.digitalocean.com/community/tutorials/how-to-set-up-nginx-server-blocks-virtual-hosts-on-ubuntu-16-04).
-
-```
-sudo tee /etc/nginx/sites-available/default <<EOF
-server {
-        listen 80 default_server;
-        listen [::]:80 default_server;
-        server_name _;
-        charset utf-8;
-
-        client_max_body_size       10m;
-        client_body_buffer_size    128k;
-
-        proxy_buffer_size          4k;
-        proxy_buffers              4 32k;
-        proxy_busy_buffers_size    64k;
-        proxy_temp_file_write_size 64k;
-
-        gzip on;
-        gzip_types
-                "application/json;charset=utf-8" application/json
-                "application/javascript;charset=utf-8" application/javascript text/javascript
-                "application/xml;charset=utf-8" application/xml text/xml
-                "text/css;charset=utf-8" text/css
-                "text/plain;charset=utf-8" text/plain;
-
-        root /srv/www/;
-        index index.html;
-
-        location /dev {
-                proxy_pass                http://localhost:8080/dev;
-                proxy_redirect            off;
-                proxy_set_header          Host               \$host;
-                proxy_set_header          X-Real-IP          \$remote_addr;
-                proxy_set_header          X-Forwarded-For    \$proxy_add_x_forwarded_for;
-                proxy_set_header          X-Forwarded-Proto  https;
-        }
-}
-EOF
-```
-
-Start the Nginx service and enable it to start on boot.
-
-```
-sudo systemctl start nginx
-sudo systemctl enable nginx
-```
-
-## Install certbot (_Optional_)
-
-[Full installation instructions for `certbot`](https://certbot.eff.org/lets-encrypt/debianbuster-nginx).
-
-```
-sudo apt-get install certbot python-certbot-nginx
-```
-
-Run the tool to setup certificates for sites configured in Nginx:
-
-```
-sudo certbot --nginx
-```
-
-> :information_source: After setting up a certificate with certbot, usually I
-> have to clean up my Nginx `default` config manually.
-
-To simulate a renewal, use:
-
-```
-sudo certbot renew --dry-run
-```
-
-For the real deal run without the `--dry-run` switch. You will want to do this
-every month or so.
 
 ## Install D2 CLI
 
```

Comment:
I think we should always just refer to the "most authoritative" source in order for us to have docs that are up-to-date.

I suggest we add a "Prerequisites" section. Maybe

```markdown
## Prerequisites

Make sure you have the following tools installed

* [Docker](https://www.docker.com/)
* [Docker Compose](https://docs.docker.com/compose/)
* [DHIS2 d2 CLI](https://cli.dhis2.nu/#/getting-started) - only needed if you want to use `d2` for setting up DHIS2 in Docker.
```

We can also move the link to `d2` at the top of a `d2` section. In case we want to split the docs into `Docker Compose` and `d2`.